backend/hct_mis_api/apps/payment/schema.py

```diff
@@ -661,24 +661,7 @@ def resolve_unsuccessful_payments_count(self, info: Any) -> int:
         return self.unsuccessful_payments_for_follow_up().count()
 
     def resolve_can_send_to_payment_gateway(self, info: Any) -> bool:
```

Comment:
in many places we use the following pattern, it's useful for type-checking
```suggestion
    @staticmethod
    def resolve_can_send_to_payment_gateway(parent: PaymentPlan, info: Any) -> bool:
```